setup.py

```diff
@@ -15,21 +14,27 @@ def install_and_compile_proto():
         print("Compiling protobuf", fn)
         compile_proto(proto_dir, dest_dir, proto_file=fn)
 
+
 class develop(_develop):
     """Post-installation for development mode."""
+
     def run(self):
         _develop.run(self)
         self.execute(install_and_compile_proto, (), msg="Compile protocol buffers")
 
+
 class install(_install):
     """Post-installation for installation mode."""
+
     def run(self):
         _install.run(self)
         self.execute(install_and_compile_proto, (), msg="Compile protocol buffers")
 
 
-with open('snet_cli/__init__.py', 'rt', encoding='utf8') as f:
-    version = re.search(r'__version__ = "(.*?)"', f.read()).group(1)
+version = {}
```

Comment:
1. What is the reason to set version to ```{}``` at line 34???
2. Is using ```exec``` is necessary here? We could use ```import``` here.

```
import snet_cli.version
version = snet_cli.version.__version__
```

Reply:
Sorry... There is an error in my previous comment. Of course you simple should do as following:

```python
from snet_cli.version import __version__ as version
```

Reply:
with
`from snet_cli.version import __version__ as version`
1. This from statement causes the ./snet_cli/__init__.py to be invoked
2. This causes a build failure as some of the dependencies would not have been set up when setup.py is run. This caused the circleci build to fail

Reply:
@astroseger Can you approve the pr if you are ok?